### PR TITLE
Added RAMA WORKS M65-B, M65-BX, M10-C, M50-A

### PR DESCRIPTION
**Pull request by Wilba6582**
## Description

Added RAMA WORKS M65-B, M65-BX, M10-C, M50-A

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wilba_tech/rama_works_m10_c/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    LAYOUT( KC_KP_7, KC_KP_8, KC_KP_9, KC_KP_4, KC_KP_5, KC_KP_6, KC_KP_1, KC_KP_2, KC_KP_3, KC_KP_0 ),
+    LAYOUT( KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS ),
+    LAYOUT( KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS ),
+    LAYOUT( KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS )
```
Comment (fauxpark) [inline keyboards/wilba_tech/rama_works_m10_c/keymaps/default/keymap.c:22]:
```suggestion
    /*LAYOUT( KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS ),
    LAYOUT( KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS ),
    LAYOUT( KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS )*/
```

Comment (Wilba6582) [reply keyboards/wilba_tech/rama_works_m10_c/keymaps/default/keymap.c:23]: OK

File: keyboards/wilba_tech/rama_works_m50_a/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ * |------+------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |             |      |      |      |      |      |
+ * `------------------------------------------------------------------------------------------'
+ */
+[3] = LAYOUT(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+)
```
Comment (noroadsleft) [inline keyboards/wilba_tech/rama_works_m50_a/keymaps/default/keymap.c:91]:
```suggestion
/*[3] = LAYOUT(
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
)*/
```

File: keyboards/wilba_tech/rama_works_m65_b/info.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "RAMA WORKS M65-B", 
+    "url": "http://rama.works", 
+    "maintainer": "Wilba", 
+    "width": 16, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```
[the remark this answers is not in the document]
Comment (Wilba6582) [reply keyboards/wilba_tech/rama_works_m65_b/info.json:8]: OK

File: keyboards/wilba_tech/rama_works_m65_bx/info.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "RAMA WORKS M65-BX", 
+    "url": "http://rama.works", 
+    "maintainer": "Wilba", 
+    "width": 16, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```
[the remark this answers is not in the document]
Comment (Wilba6582) [reply keyboards/wilba_tech/rama_works_m65_bx/info.json]: OK

File: keyboards/wilba_tech/rama_works_m50_a/keymaps/default/keymap.c (hunk 2 above)
Comment (Wilba6582) [reply keyboards/wilba_tech/rama_works_m50_a/keymaps/default/keymap.c:92]: OK

**Comment by Wilba6582** [comment]
Hold up, I will change the number of layers for M10-C because someone just asked for more

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**